vllm/entrypoints/openai/api_server.py

```diff
@@ -386,6 +386,24 @@ async def get_server_load_metrics(request: Request):
     return JSONResponse(content={"server_load": request.app.state.server_load_metrics})
 
 
+
+@router.get("/get_server_info")
```

Comment:
Is this needed? I think its usually not great to allow external users to see this type of server info

I think that instead we should make sure the server is in DEV mode to expose this

Reply:
I agree it's good to keep the discovery part separate (can figure out in future PR). The header changes in this one look good to me now.